src/rdo.rs

```diff
@@ -432,13 +455,46 @@ pub fn rdo_tx_type_decision(
 
   let cw_checkpoint = cw.checkpoint();
 
+  let mut mc_block: AlignedArray<[[u16; 64 * 64]; 3]> = UninitializedAlignedArray();
+
+  // If inter mode, save motion compensated pixles since it will be overriden by write_tx_tree()
+  if is_inter {
+    motion_compensate(fi, fs, cw, mode, ref_frame, mv, bsize, bo, bit_depth);
```

Comment:
Might be interesting to have `motion_compensate` take a destination slice so you'd spare the copy.